src/renderer/themes.css

```diff
@@ -964,6 +1080,90 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --primary-color-active: #8a8abf;
 }
 
+.mainEverforestDarkRed {
+  --primary-color: #e67e80;
+  --primary-color-hover: #e89e9f;
+  --primary-color-active: #d65f61;
+}
+
```

Comment:
**issue (blocking):** PR is missing CSS blocks for the new main light and dark colors, causing stuff like this to happen.

![Screenshot_20241230_151238](https://github.com/user-attachments/assets/13bb66eb-aff1-4022-987e-2e9f030b5c8e)

Blocks from the Solarized theme colors for reference. You can also then mostly copy your created solution to the secondary theme colors by using `sec` in the names intead of `main`:


```
.mainSolarizedYellow,
.mainSolarizedOrange,
.mainSolarizedRed,
.mainSolarizedMagenta,
.mainSolarizedViolet,
.mainSolarizedGreen {
  --text-with-main-color: #fdf6e3;
  --logo-icon-bar-color: url('../../_icons/iconDraculaLightSmall.svg');
  --logo-text-bar-color: url('../../_icons/textDraculaLightSmall.svg');
}

.mainSolarizedBlue,
.mainSolarizedCyan {
  --text-with-main-color: #000;
  --logo-icon-bar-color: url('../../_icons/iconDraculaDarkSmall.svg');
  --logo-text-bar-color: url('../../_icons/textDraculaDarkSmall.svg');
}
```